app/src/main/java/com/android/nikhil/worldnow/ui/news/NewsViewModal.kt

```diff
@@ -3,28 +3,38 @@ package com.android.nikhil.worldnow.ui.news
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
 import android.util.Log
+import android.widget.Toast
 import com.android.nikhil.worldnow.BuildConfig
+import com.android.nikhil.worldnow.data.model.main_news.Result
+import com.android.nikhil.worldnow.data.source.datasource.NewsDataSource
+import com.android.nikhil.worldnow.data.source.repository.NewsRepository
 import com.android.nikhil.worldnow.network.NewsInterface
-import com.android.nikhil.worldnow.utils.MainResponse
-import com.android.nikhil.worldnow.utils.Result
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.schedulers.Schedulers
 import retrofit2.Call
 import retrofit2.Callback
+import java.util.*
 import javax.inject.Inject
 
 class NewsViewModal @Inject constructor(): ViewModel() {
```

Comment:
Since you've touched this file, please rename this file to `NewsViewModel` to fix the typo.